advocacy_docs/kubernetes/cloud_native_operator/expose_pg_services.mdx

```diff
@@ -8,7 +8,7 @@ This section explains how to expose a PostgreSQL service externally, allowing ac
 to your PostgreSQL database **from outside your Kubernetes cluster** using
 NGINX Ingress Controller.
 
-If you followed the [QuickStart](/quickstart), you should have by now
+If you followed the [QuickStart](./quickstart.md), you should have by now
```

Comment:
Ensure link works, as per #1009, #1029

Reply:
@cero-miedo This being a relative link should work just fine, right? The problem was that it was an absolute top-level link before?

Reply:
Tested locally, works fine. Thanks for following up.